README.md

```diff
@@ -80,6 +92,51 @@ If you prefer the command line way of downloading and installing things, then fe
   - The page will reload if you make edits.
   - You will also see any lint errors in the console.
 
+- Insert habit dummy data:
+
+  - Go back to the MongoDB website to your project.
+  - The collection with the databasename you provided will be present (created when you started the server app)
+  - Inside the collection, you will find 'habits' as a subcollection
+  - Open it and you can insert the below dummy documents
+
+```
+{
+  category: "Recycle",
+  description: "Reduce the amount of waste by using reusable bags!",
+  question: "Which one of these did you use today?",
+  answerOptions: ["clothbag"],
+  __v:0
+}
+```
+
+```
+{
+  category: "Commute",
+  description: "Carpool with coworker or friends",
+  question: "How did you commute to work today?",
+  answerOptions: ["Uber", "Lyft"],
+  __v:0
+}
+```
+
+```
+{
+  category: "Recycle",
+  description: "Separate bin for recycle stuff",
+  question: "How much did you dump today for recycle",
+  answerOptions: ["3boxes", "4boxes"],
+  __v:0
+}
+```
```

Comment:
Just paid attention to the content here, I know this is just test data, but I think it would be nice to have the data closer to something we would really use in real. For example, having Uber or Lyft as ridesharing is not something our habit tracker. See the below data as example: 

```jason
  {
    "category": "Recycle",
    "description": "Reduce the amount of waste by using reusable bags!",
    "question": "Which one of these did you use today?",
    "answerOptions": ["Reusable bag", "Plastic bag", "Both", "None"]
  }
```
```jason
  {
    "category": "Commute",
    "description": "Carpool with coworker or friends",
    "question": "How did you commute to work today?",
    "answerOptions": ["Drived", "Carpooled", "Bike", "Public Transport"]
  }
```

```jason
  {
    "category": "Recycle",
    "description": "Separate your recycble materials",
    "question": "Did you separate your recycble materials today?",
    "answerOptions": ["Yes", "No"]
  }
```